lib/rules/no-only-experiments.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
         //----------------------------------------------------------------------
 
         function ifExperimentReport(name, node) {
-            if(name == 'experiment' || name === 'describe') {
+            if(name == 'experiment' || name === 'describe'|| name === 'test' ) {
```

Comment:
Lab has `it` as an alias of `test` similar to how `describe` is an alias of `experiment`.